salt/renderers/gpg.py

```diff
@@ -373,7 +373,7 @@ def _decrypt_ciphertexts(cipher, translate_newlines=False):
         return ret.rstrip('\n')
     else:
         # Possibly just encrypted data without begin/end marks
```

Comment:
Do we need this comment here now?

Reply:
Seems like this should be removed.

Reply:
@secumod yes, you're right. Thank you for review.